influxdb_client/client/influxdb_client_async.py

```diff
@@ -189,8 +191,8 @@ def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gz
             }
 
         """
-        return InfluxDBClientAsync._from_config_file(config_file=config_file, debug=debug,
-                                                     enable_gzip=enable_gzip, **kwargs)
+        return InfluxDBClientAsync._from_config_file(config_file=config_file, config_name=config_name,
+                                                     debug=debug, enable_gzip=enable_gzip, **kwargs)
```

Comment:
Please use `kwargs` instead of new argument:
```suggestion
        return InfluxDBClientAsync._from_config_file(config_file=config_file, debug=debug, enable_gzip=enable_gzip, **kwargs)
```